Pcap++/header/DpdkDevice.h

```diff
@@ -12,45 +12,48 @@
+ * PcapPlusPlus framework and API so you can use DPDK together with other PcapPlusPlus features such as packet parsing
+ * and editing, etc.<BR> So how DPDK basically works? in order to boost packet processing performance on a commodity
+ * server DPDK is bypassing the Linux kernel. All the packet processing activity happens in the user space so basically
+ * packets are delivered from NIC hardware queues directly to user-space shared memory without going through the kernel.
+ * In addition DPDK uses polling instead of handling interrupts for each arrived packet (as interrupts create some
+ * delays). Other methods to boost packets processing implemented by DPDK are using Hugepages to decrease the size of
+ * TLB that results in a much faster virtual to physical page conversion, thread affinity to bind threads to a specific
+ * core, lock-free user-space multi-core synchronization using rings data structures and NUMA awareness to avoid
+ * expensive data transfers between sockets.<BR> Not every NIC supports kernel-bypass capabilities so DPDK cannot work
+ * with any NIC. The list of supported NICs are in DPDK's web-site http://dpdk.org/doc/nics . For each such NIC the DPDK
+ * framework provides a module that called poll-mode-driver (PMD in short) that enables this NIC to the working with
+ * DPDK. PcapPlusPlus wasn't tested with most PMDs but all of them should theoretically work as PcapPlusPlus doesn't
+ * change the PMD behavior<BR> DPDK has another basic data-structure called mbuf. An mbuf is DPDK wrapper struct for
+ * network packets. When working with packets in DPDK you actually work with mbufs. The mbuf contains the packet data
+ * (obviously) but also some metadata on the packet such as the DPDK port it was captured on, packet ref-count (which
+ * allows it to be referenced by several objects), etc. One important concept is that DPDK doesn't allocate mbufs
+ * on-the-fly but uses mbuf pools. These pools is allocated on application startup and used throughout the application.
+ * The goal of this, of course, is increasing packet processing performance as allocating memory has its cost. So pool
+ * size is important and varies between applications. For example: an application that stores packets in memory has to
+ * have a large pool of mbufs so mbufs doesn't run-out. PcapPlusPlus enables to choose the pool size at startup<BR> <BR>
  * PcapPlusPlus main wrapper classes for DPDK are:
- *    - DpdkDevice - a class that wraps a DPDK port and provides all capabilities of receiving and sending packets to this port
- *    - DpdkDeviceList - a singleton class that initializes the DPDK infrastructure and creates DpdkDevice instances to all available ports.
- *      In addition it allows starting and stopping of worker threads
+ *    - DpdkDevice - a class that wraps a DPDK port and provides all capabilities of receiving and sending packets to
+ * this port
```

Comment:
Let's add a few spaces to align with the bullet point in the previous line?
Same in lines:
- 46
- 49-51
- 56

Reply:
Should be fixed in bf2631d